nominees/okulab.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "OKULAB",
+  "aliases": [
+    "https://okulab.kz/"
+  ],
+  "description": "Platform for the ",
+  "website": "https://okulab.kz/",
+  "license": [
+    {
+      "spdx": "OSL-1.0",
```

Comment:
A software license was provided instead of an open content license. Please check with submitter. Thanks.